### DISCUSSION TITLE
error while opening Content.mgcb

### DISCUSSION THREAD

**Discussion by Kubrix** [MonoGame/MonoGame#8206] [General]
Windows 10x64
Studio 2022 pro
MonoGame Cross Platform Desktop Project templates installed
.NET 6.0 SDK (v6.0.419) installed

Create "MonoGame Cross Platform Desktop Project." project
When clicking Content-Content.mgcb after some time i see error text

System.ArgumentNullException: Value can not be undefined
Parameter: docView
   в Microsoft.Requires.NotNull[T](T value, String parameterName)

**Comment by BlizzCrafter** [comment]
The problem seems to be coming from the Visual Studio extension which normally opens the pipeline tool.

I found on [this line](https://github.com/MonoGame/MonoGame/blob/develop/Templates/MonoGame.Templates.VSExtension/MonoGameTemplatesVSExtensionPackage.cs#L67) that the `ppunkDocView` value gets initialized to zero. This should be fine, because the pipeline tool is an external tool and should not be opened in a pane directly inside Visual Studio ([ref](https://learn.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.shell.interop.ivseditorfactory.createeditorinstance?view=visualstudiosdk-2022#microsoft-visualstudio-shell-interop-ivseditorfactory-createeditorinstance(system-uint32-system-string-system-string-microsoft-visualstudio-shell-interop-ivshierarchy-system-uint32-system-intptr-system-intptr@-system-intptr@-system-string@-system-guid@-system-int32@))).

However, I recently installed MonoGame and the VSExtension on a fresh pc and didn't encounter this problem myself. The whole installation process worked as described in the docs.

I see that you are using the pro version of Visual Studio 2022, but I don't know if this really makes a difference, because I can't test this myself.

Probably switch to the community edition of VS2022 and see if this makes a difference.
